extensions/amp-story-auto-ads/0.1/amp-story-auto-ads-attribution.css

```diff
@@ -1,9 +1,7 @@
-
-
 .i-amphtml-story-ad-attribution {
   position: absolute;
-  bottom: 0 !important;
-  left: 0 !important;
+  bottom: 10px !important;
+  right: 10px !important;
```

Comment:
Just curious, you're not switching the badge to the other side of the page for RTL layouts?

Reply:
It appears that we have not been doing that before (I don't know the right answer. Is it better or worse?). This change you pointed out is the adchoice icon.